containerize/Containerfile

```diff
@@ -40,9 +35,10 @@ FROM registry.access.redhat.com/ubi9-minimal
 COPY --from=deps /opt/zap /opt/zap
 COPY --from=deps /opt/firefox /opt/firefox
 COPY --from=deps /usr/local/bin/kubectl /usr/local/bin/kubectl
-COPY --from=deps /tmp/trivy/trivy /usr/local/bin/trivy
 COPY --from=deps /tmp/redocly/node_modules /opt/redocly/node_modules
 
+RUN rpm -ivh https://github.com/aquasecurity/trivy/releases/download/v0.54.1/trivy_0.54.1_Linux-64bit.rpm
```

Comment:
The two stage build is used to reduce the image size. Can this be improved to reduce size too?